test/predcorr_tests.jl

```diff
@@ -0,0 +1,89 @@
+using StochasticDiffEq, DiffEqProblemLibrary, DiffEqDevTools, Base.Test
```

Comment:
Just rename this file as `nondiagonal_tests.jl`: I assume other algorithms will get added to it later.

Reply:
Will do.